### DISCUSSION TITLE
possibe to track changes for rasters efficiently?

### DISCUSSION THREAD

**Discussion by wiesehahn** [koordinates/kart#976] [Q&A]
If I would use kart to track changes in a raster data set (e.g. DEM), does is push the entire raster file to LFS for each version or is it able to just store the changes?

**Comment by rcoup** [comment]
Hi,

> does [Kart] push the entire raster file to LFS for each version or is it able to just store the changes?

It pushes the entire raster file for each version, but does de-duplicate tiles with the same content & different names.

We've had some ideas floating around on de-duplicating blocks on the server, or using compressed arrays as deltas, but we're more interested in seeing how it's used first.
